packages/cli/src/commands/__tests__/cli-commands.test.ts

```
@@ -33,7 +33,7 @@ describe("M3 CLI command IA", () => {
     const names = cli.commands.map((command) => command.name());
 
     expect(names).toEqual(
-      expect.arrayContaining(["run", "test", "plugin", "audit", "policy", "init", "quickstart"])
+      expect.arrayContaining(["run", "test", "plugin", "audit", "policy", "init", "quickstart", "models"])
```

ESLint

[error] Replace `"run",·"test",·"plugin",·"audit",·"policy",·"init",·"quickstart",·"models"` with `⏎········"run",⏎········"test",⏎········"plugin",⏎········"audit",⏎········"policy",⏎········"init",⏎········"quickstart",⏎········"models",⏎······`

(prettier/prettier)